File: void-lib/void-separate-modules/void-common/src/main/kotlin/cn/toutatis/xvoid/common/exception/UnableToResolvedException.java
package cn.toutatis.xvoid.common.exception;

/**
 * @author Touatatis_Gc
 * 无法找到jsonobject对象key的异常
 */
public class UnableToResolvedException extends RuntimeException {

    /**
     * Constructs a {@code UnableToResolvedException} with no detail message.
     */
    public UnableToResolvedException() {
    }

    /**
     * Constructs a {@code UnableToResolvedException} with the specified
     * detail message.
     *
     * @param   message   the detail message.
     */
    public UnableToResolvedException(String message) {
        super(message);
    }
}

File: void-lib/void-common/src/main/kotlin/cn/toutatis/xvoid/common/exception/UnableToResolvedException.kt
package cn.toutatis.xvoid.common.exception

import java.lang.RuntimeException

/**
 * @author Touatatis_Gc
 * 无法找到jsonobject对象key的异常
 */
class UnableToResolvedException : RuntimeException {
    /**
     * Constructs a `UnableToResolvedException` with no detail message.
     */
    constructor() {}

    /**
     * Constructs a `UnableToResolvedException` with the specified
     * detail message.
     *
     * @param   message   the detail message.
     */
    constructor(message: String?) : super(message) {}
}